README.md

```diff
@@ -21,8 +21,8 @@
 
 ## What is tldr-pages?
 
-The **tldr-pages** project is a collection of community-maintained help pages
-for command-line tools, that aims to be a simpler, more approachable complement
+The **tldr-pages** project is the biggest collection of community-maintained help pages
```

Comment:
Personal Opinion:

I would generally err on the side of `humility` and not add an adjective :) [Although, I agree this is statistically maybe true afterall]

Reply:
I suggested it because in my opinion too much framentation is bad. This statistic fact helps to centralize the efforts in a single project if they think tldr is good enough fot their needs.

Reply:
There is even a "similar projects" section, IMO the general idea of the README still remains being the open source collaboration, and not competition.

Reply:
In agreement to your points; I gave my opinion :)
(not wearing the project collaborator hat - but a personal opinion)

Reply:
I fully agree with @mfrw. It's not a competition - I personally wouldn't add this.

Reply:
It's OK if we decide to not merge this, but my point is that this is not for saying that tldr is good and other similar projects are worse, but just that tldr attracted a big attention from the community. This can help users to choose what project they will use.

But, more importantly, this fact emphasizes the community **colaboration**. BTW, tldr can be used as a base for another similar project that can eventually become even bigger than tldr, highlighting again the **collaboration** among the open source community, and that wouldn't be bad, but in fact awesome, so we could remove this adjective.

But, if you think this could be misinterpreted, we can decide to not merge this.

Reply:
I'm not going to vehemently stand against merging this - after all, it's just one adjective, not a huge banner that says "we're the best". That being said, my opinion hasn't changed. Go ahead and request more reviews if you want to discuss it.

Reply:
Sure, it would be good to know the opinion of more people